python/cugraph/cugraph/tests/centrality/test_betweenness_centrality.py

```diff
@@ -118,7 +118,7 @@ def calc_betweenness_centrality(
     )
 
     M = G.to_pandas_edgelist().rename(
-        columns={"src": "0", "dst": "1", "weights": "weight"}
+        columns={"src": "0", "dst": "1", "wgt": edge_attr}
```

Comment:
this is another place where I would like to see a references.  We have switched from "wt" to "weight" to "weights", and now to "wgt".   by using a reference we would only change the column names in one place

Reply:
When using the datasets API to get a graph with `graph_file.get_graph`, this implicitly has the column names hardcoded to 'src', 'dst', and 'wgt'. For instance the [karate](https://github.com/rapidsai/cugraph/blob/branch-23.08/python/cugraph/cugraph/datasets/metadata/karate.yaml#L13:L15) datasets in the yaml file